src/Core/Logger/Util/Introspection.php

```diff
@@ -86,6 +87,7 @@ public function getRecord(): array
 			'line'       => $trace[$i - 1]['line'] ?? null,
 			'function'   => $trace[$i]['function'] ?? null,
 			'request-id' => $this->requestId,
+			'stack'      => System::callstack(10, 4),
```

Comment:
You should also set the `$full` parameter to true to capture every function call. Without this, the system skips all function call in the same class.

Reply:
Only for the database calls, which isn't what I'm trying to capture here.

Reply:
Well, it doesn't cause any harm either.

Reply:
If enough Database calls are stringed together, it would eat up most of the limit of 10 calls. Since the database connector is pretty well tested, I don't think it should be included in this particular call stack.

Reply:
Okay, let's gain some experience with this.